### PR TITLE
Ctpdev:digi2raw and ctp-raw-recoder fixes

### PR DESCRIPTION
**Pull request by lietava**
Fixes described here:
https://github.com/AliceO2Group/AliceO2/pull/7121#issue-1006855566
:
CTP decoding was producing more logs than all other detectors together, so I've changed a lot of INFOs to DEBUG in your code.
Also, had to touch your digits class.

The CTP digit2row still has this custom option --file-per-link instead of --file-for [all,link,cru], could you please fix this? The output naming is ok.

Then, your raw->digits decoder is for some reason named o2-ctp-reco-workflow, Could you please rename it to something more appropriate, e.g. o2-ctp-raw-decoder. More serious is that it declares a lot of options which it is not supposed to support, I guess it comes from some copy/paste.

  --input-type arg (=raw)               digits, raw
  --output-type arg (=digits)           digits, raw
  --disable-mc                          disable sending of MC information
  --disable-root-input                  disable root-files input reader
  --disable-root-output                 disable root-files output writer
Can you have input other than raw and output other than digits?
Raw decoders are not supposed to support MC, so you don't need this option at all.
And for the raw decoders there is no root input anyway.

### CODE REVIEW

**Comment by lietava** [comment]
@shahor02 please, have a look and let me know if I understood correctly.

**Review by shahor02 — DISMISSED** [review]

**Comment by davidrohr** [comment]
@lietava : this fails in the full system test with
```
## sw/BUILD/O2-full-system-test-latest/log
--
command o2-ctp-digi2raw -o raw/CTP --file-per-link had nonzero exit code 1
----- Last log: -----
/sw/BUILD/f1e60ca2e640e01e5f09722b7f71455a79be460f/O2-full-system-test/full-system-test-sim
Running: TIME="#walltime %e" /sw/slc8_x86-64/O2/7472-slc8_x86-64-local1/share/scripts/monitor-mem.sh /usr/bin/time --output=ctpraw.log_time './ctpraw.log_tmp.sh'
/usr/bin/time --output=ctpraw.log_time ./ctpraw.log_tmp.sh
ERROR: unrecognised option '--file-per-link'
```

could you have a look?

**Comment by shahor02** [comment]
@lietava could you change in the line  https://github.com/lietava/AliceO2/blob/2517903f47b611e0babb12da3cefe41499f1c722/prodtests/full_system_test.sh#L133
`-file-per-link` to `--file-for cru`
and commit it to this PR?

**Comment by lietava** [comment]
full_system_test --file-for cru done

**Comment by shahor02** [comment]
Hi @lietava 
could you rebase your branch to fresh dev and force-push? The FST in the FullCI failed with strange error from the dpl-workflow, but for me the dev works fine.

**Comment by lietava** [comment]
@shahor02 dev rebased to ctpdev and pushed

File: Detectors/CTP/workflow/src/ctp-raw-decoder.cxx [hunk 1]
```diff
@@ -29,12 +29,7 @@ void customize(std::vector<o2::framework::ConfigParamSpec>& workflowOptions)
 {
   std::vector<o2::framework::ConfigParamSpec> options{
     {"input-type", o2::framework::VariantType::String, "raw", {"digits, raw"}},
-    {"output-type", o2::framework::VariantType::String, "digits", {"digits, raw"}},
-    {"disable-mc", o2::framework::VariantType::Bool, false, {"disable sending of MC information"}},
-    {"disable-root-input", o2::framework::VariantType::Bool, false, {"disable root-files input reader"}},
-    {"disable-root-output", o2::framework::VariantType::Bool, false, {"disable root-files output writer"}},
-    {"configKeyValues", o2::framework::VariantType::String, "", {"Semicolon separated key=value strings ..."}},
```
Comment (davidrohr) [inline Detectors/CTP/workflow/src/ctp-raw-decoder.cxx:36]: This option `configKeyValues` must remain

**Review by davidrohr — CHANGES_REQUESTED** [review]
The FullCI is failing, since you have removed the configKeyValues options (see below). Could you add that again please?

**Comment by lietava** [comment]
@davidrohr configKeyValues is back

**Pull request merged**